dev-system/project.clj

```diff
@@ -53,6 +53,7 @@
                [ring/ring-codec]]
   :dependencies ~(concat '[[commons-codec/commons-codec "1.11"]
                            [org.clojure/clojure "1.10.0"]
+                           ;; TODO Should these keep up with other dependencies?
```

Comment:
Will remove if someone let's me know. They were not previously kept up with specifically the `ring-codec`